freqtrade/configuration/config_validation.py

```diff
@@ -323,6 +324,14 @@ def _validate_pricing_rules(conf: Dict[str, Any]) -> None:
             del conf['ask_strategy']
 
 
+def _validate_freqai_hyperopt(conf: Dict[str, Any]) -> None:
+    freqaimodel = conf.get('freqaimodel')
```

Comment:
i think you should validate `conf.get('freqai', {}).get('enabled')` here ... checking on freqaimodel is ... odd - to say the least

Reply:
You are right! Fixed.